apps/condo/domains/common/constants/currencies.js

```diff
@@ -181,11 +180,15 @@ const ISO_CODES = [
 ]
 
 const ISO_CODES_FOR_SBBOL = {
-    '643': 'RUB',
-    '810': 'RUR',
+    '810': 'RUB',
     '840': 'USD',
     '978': 'EUR',
 }
+
+const ALTERNATIVE_CURRENCY_CODES_FROM_SBBOL = {
```

Comment:
I'm not sure this one should be place here....

Why `SBBOL` in common and on open-source part?

Reply:
> Why SBBOL in common ...

You're right, it shouldn't be here. Moved it to integration folder.

> Why SBBOL ... on open-source part?

This is how it happened historically. I won't change anything in this regard in this pr :)